app/views/browse/route.html

```diff
@@ -204,7 +204,12 @@ <h4 class="mar-top-xl">Details</h4>
                       <pre class="clipped">{{route.spec.tls.destinationCACertificate}}</pre>
                     </div>
                   </dl>
-                  <div ng-if="!route.spec.tls"><em>TLS is not enabled for this route</em></div>
+                  <p ng-if="!route.spec.tls">
+                    TLS is not enabled.
+                    <span ng-if="'routes' | canI : 'update'">
+                      <a ng-href="{{route | editResourceURL}}" role="button">Edit</a> this route to enable security.
```

Comment:
maybe "Edit this route to enable secure network traffic" or something instead of just "security"